specification/2.0/README.md

```diff
@@ -3901,9 +3889,49 @@ Application-specific data.
 
 # Appendix B: BRDF Implementation
 
-*This section is non-normative.*
+The metallic-roughness material model is a linear blend of two BRDFs, a metallic BRDF and a dielectric BRDF. The blending factor `metallic` describes the metalness of the material. The BRDFs share the parameters for roughness and base color, following [Burley (2012): Physically-Based Shading at Disney](https://disney-animation.s3.amazonaws.com/library/s2012_pbs_disney_brdf_notes_v2.pdf).
+
+```
+material = mix(dielectric_brdf, metal_brdf, metallic)
+```
+
+The metal BRDF is based on a microfacet model which describes the orientation of microfacets on the surface as a statistical distribution. The distribution is controlled by a parameter called `roughness`, varying between 0 (smooth surface) and 1 (rough surface). As the blending between the two extremes does not behave linearly, like most microfacet distributions, the material's `roughness` parameter is squared before using it as the distribution's roughness.
```

Comment:
I would describe the microfacet model separately since it is used by both metals and dielectrics. It is the Trowbridge-Reitz model (also known as GGX) where the microfacets are distributed on infinitesimal oblate ellipsoids, whose half-height in the normal direction is the `roughness^2` times the radius in the tangent plane. Roughness == 1 gives spheres, which result in uniform reflection in all directions, absent geometric shadowing. Geometric shadowing and multiscattering can be derived from this microfacet distribution, with common approximations given in the implementation section.

Reply:
Thanks a lot for that! I added it to the text.